source/secrets/index.rst

```diff
@@ -90,10 +90,10 @@ Dans le cas de OAuth, la clé de signature est composée de la clé secrète (ab
 
 Le processus de récupération des données à une API est le suivant:
 
-- 1 On envoie au serveur de vérification le message de requête comprenant la clé d'utilisateur. À celui-ci est ajouté le hash généré à partir du message et de la clé de signature (c'est la que :py:mod:`hmac` est utile). Cette opération est effectuée à chaque fois que le client essaye de se connecter.
-- 2 Si le destinataire du message est vérifié et que le compte est autorisé, le serveur renvoie à l'application un jeton.
-- 3 L'application envoie ce jeton au serveur de ressources qui vérifie grace à celui-ci si l'application est autorisée, et ses privilèges.
-- 4 Si l'application est autorisée, le serveur renvoie les informations requises.
+#. On envoie au serveur de vérification le message de requête comprenant la clé d'utilisateur. À celui-ci est ajouté le hash généré à partir du message et de la clé de signature (c'est la que :py:mod:`hmac` est utile). Cette opération est effectuée à chaque fois que le client essaye de se connecter.
+#. Si le destinataire du message est vérifié et que le compte est autorisé, le serveur renvoie à l'application un jeton.
+#. L'application envoie ce jeton au serveur de ressources qui vérifie grace à celui-ci si l'application est autorisée, et ses privilèges.
+#. Si l'application est autorisée, le serveur renvoie les informations requises.
```

Comment:
https://dev.twitter.com/web/sign-in/implementing

En fait c'est encore bien plus compliqué apparement